rest-client-reactive-quickstart/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,54 @@
+####
```

Comment:
We could drop the docker files if they are not referenced in the guide.

Reply:
how strong do you feel on this Clement? I think they bring some value here, even if they are not mentioned...

Reply:
I don't mind keeping them. It's just a pain when we want to add a new one, or when ubi needs to be updated.

Reply:
done